src/components/atoms/Selector/SelectorList/SelectorListItem/SelectorListItem.tsx

```diff
@@ -8,18 +8,21 @@ export const SelectorListItem = <OptionValue extends BaseOptionValue>({
   option,
   isActive,
   onClick,
+  disabled,
```

Comment:
[imo nits]
初期値でfalseを指定しても良いかもです！
```suggestion
  disabled = false,
```

Reply:
修正しました。